### ISSUE TITLE
Error when using it with JSPM

### ISSUE THREAD

**Issue by radenkozec** [istrau2/aurelia-flatpickr#3]
**I'm submitting a bug report**

**Current behavior:**
_**On JSPM I am getting this error when try to call it with defaults:**_
aurelia-flatpickr.js:99 Uncaught (in promise) TypeError: _flatpickr2.default is not a constructor
    at AureliaFlatpickrCustomElement.attached (aurelia-flatpickr.js:99)
    at Controller.attached (aurelia-templating.js:3513)
    at View.attached (aurelia-templating.js:1563)
    at Controller.attached (aurelia-templating.js:3517)
    at DialogRenderer.attach (dialog-renderer.js:132)
    at DialogRenderer.showDialog (dialog-renderer.js:208)
    at eval (dialog-service.js:78)
    at <anonymous>

**Comment by istrau2** [comment]
Hey,

Just looking at the error, it seems that you have not included the actual flatpickr library (js and css file) in your project. Could that be your issue?

**Comment by radenkozec** [comment]
@istrau2 Not quite. Actually I have installed it using JSPM install NPM:flatpickr. I guess that you have route to the flatpickr library hardcoded for WebPack and node_modules folder and my library actually is installed in jspm_packages/npm folder when using JSPM.

**Comment by istrau2** [comment]
Ah, ok. I understand the issue. Well, this feels like something that needs to be solved in the bundler. For example, I know in webpack, you can alias a file so that 'flatpickr' resolves to '/path/to/flatpickr'. Do you have some other idea?

**Comment by PhilippBaranovskiy** [comment]
I've got the same issues
